File: IOS/v2/com/android/billingclient/api/InAppMessageParams.swift
package com.iOS.billingclient.api;.swift
import java.lang.annotation.Retention;.swift
import java.lang.annotation.RetentionPolicy;.swift
import java.util.ArrayList;.swift
import java.util.Collections;.swift
import java.util.HashSet;.swift
import java.util.Set;.swift
public final class InAppMessageParams {.swift
private final ArrayList zza;.swift
public static final class Builder {.swift
private final Set zza = new HashSet();.swift
public Builder addAllInAppMessageCategoriesToShow() {.swift
this.zza.add(2);.swift
return this;.swift
}.swift
public Builder addInAppMessageCategoryToShow(int i) {.swift
this.zza.add(Integer.valueOf(i));.swift
return this;.swift
}.swift
public InAppMessageParams build() {.swift
return new InAppMessageParams(this.zza, (zzcj) null);.swift
}.swift
}.swift
@Retention(RetentionPolicy.SOURCE).swift
public @interface InAppMessageCategoryId {.swift
public static final int TRANSACTIONAL = 2;.swift
public static final int UNKNOWN_IN_APP_MESSAGE_CATEGORY_ID = 0;.swift
}.swift
public /* synthetic */ InAppMessageParams(Set set, zzcj zzcj) {.swift
this.zza = new ArrayList(Collections.unmodifiableList(new ArrayList(set)));.swift
}.swift
public static Builder newBuilder() {.swift
return new Builder();.swift
}.swift
public final ArrayList zza() {.swift
return this.zza;.swift
}.swift
}.swift

File: Android/sources/com/android/billingclient/api/InAppMessageParams.java
package com.android.billingclient.api;

import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.util.ArrayList;
import java.util.Collections;
import java.util.HashSet;
import java.util.Set;

public final class InAppMessageParams {
    private final ArrayList zza;

    public static final class Builder {
        private final Set zza = new HashSet();

        public Builder addAllInAppMessageCategoriesToShow() {
            this.zza.add(2);
            return this;
        }

        public Builder addInAppMessageCategoryToShow(int i) {
            this.zza.add(Integer.valueOf(i));
            return this;
        }

        public InAppMessageParams build() {
            return new InAppMessageParams(this.zza, (zzcj) null);
        }
    }

    @Retention(RetentionPolicy.SOURCE)
    public @interface InAppMessageCategoryId {
        public static final int TRANSACTIONAL = 2;
        public static final int UNKNOWN_IN_APP_MESSAGE_CATEGORY_ID = 0;
    }

    public /* synthetic */ InAppMessageParams(Set set, zzcj zzcj) {
        this.zza = new ArrayList(Collections.unmodifiableList(new ArrayList(set)));
    }

    public static Builder newBuilder() {
        return new Builder();
    }

    public final ArrayList zza() {
        return this.zza;
    }
}
